examples/MuttsVersionedTrees/MuttsVersionedTrees.jl

```diff
@@ -125,7 +126,7 @@ VTreeNode{Int64}(5) 🔒
 """
 Base.insert!(::EmptyVTree{T}, x) where T = VTreeNode{T}(x)
 function Base.insert!(head::VTreeNode{T}, x) where T
-    head = getmutableversion(head)
+    head = get_mutable_version(head)
```

Comment:
Should I change to `mutable_version`, suggested in #10?

Reply:
👍 Yeah, that makes sense! I'm still unsure about what is better. I wish there was a standard that we could defer to here.... but yeah, i'd say go ahead and switch it, given @comnik's stated preference.

Reply:
Done